v0/sources/signl4/transform.js

```diff
@@ -0,0 +1,90 @@
+const path = require("path");
+const fs = require("fs");
+const { flattenJson, removeUndefinedAndNullValues } = require("../../util");
+const Message = require("../message");
+
+// import mapping json using JSON.parse to preserve object key order
+const mapping = JSON.parse(
+  fs.readFileSync(path.resolve(__dirname, "./mapping.json"), "utf-8")
+);
+
+function settingProperties(event, message) {
+  const flatEvent = event;
+  const messageReplica = message;
```

Comment:
No need to use another variable. We can directly operate on `message` and `event`.

Reply:
Removed the flatEvent one but for the message, eslint was disallowing the assignment to the function parameter. That's why create a replica for this.